lnbits/extensions/boltcards/lnurl.py

```diff
@@ -43,26 +43,26 @@
 ###############LNURLWITHDRAW#################
 
 # /boltcards/api/v1/scan?p=00000000000000000000000000000000&c=0000000000000000
-@boltcards_ext.get("/api/v1/scan/{card_uid}")
-async def api_scan(p, c, request: Request, card_uid: str = None):
+@boltcards_ext.get("/api/v1/scan/{card_id}")
+async def api_scan(p, c, request: Request, card_id: str = None):
     # some wallets send everything as lower case, no bueno
     p = p.upper()
     c = c.upper()
     card = None
     counter = b""
     try:
-        card = await get_card_by_uid(card_uid)
-        card_uid, counter = decryptSUN(bytes.fromhex(p), bytes.fromhex(card.k1))
+        card = await get_card(card_id)
+        card_id, counter = decryptSUN(bytes.fromhex(p), bytes.fromhex(card.k1))
 
-        if card.uid.upper() != card_uid.hex().upper():
+        if card.uid.upper() != card_id.hex().upper():
```

Comment:
If we're dropping card_uids entirely, we can't verify the card_uid through the encryption.